samples/net/cloud/aws_iot_mqtt/boards/ek_ra8m1.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&eth {
+	local-mac-address = [00 11 22 33 44 56];
+	status = "okay";
+	phy-handle = <&phy>;
+};
+
+&mdio {
+	pinctrl-0 = <&ether_default>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	phy: ethernet-phy@5 {
+		compatible = "ethernet-phy";
+		reg = <5>;
+		status = "okay";
+	};
+};
+
```

Comment:
@maass-hamburg , the reason I add overlay for this instead of define on board dts is because for using ethernet on EK RA8M1 board, user need to remove Jumper J61 so Ethernet is not a default on this board original state

Reply:
If it is part of the board it should be part of the boards dts, even if it  is not activated. You can mention that and what is needed to enable it in a comment in the dts and in the boards doc. BY the way if the board docs here on zephyr say "Ethernet support", it is expected that ethernet works without having a overlay.

Or is there some interference with other parts?

Reply:
@maass-hamburg , thank you, I got it. I'll move ethernet node in board dts and add a note in document on how to enable the Ethernet on the board